packages/babel-plugin-transform-runtime/src/index.js

```diff
@@ -8,28 +8,83 @@ export default declare((api, options) => {
   api.assertVersion(7);
 
   const {
-    helpers,
-    moduleName = "@babel/runtime",
-    polyfill,
-    regenerator,
-    useBuiltIns,
-    useESModules,
+    corejsVersion = false,
```

Comment:
Maybe this option should be named just `corejs`? `corejs: 2` looks as good as `corejsVersion: 2` to me, but `corejs: false` looks better than `corejsVersion: false`.

Reply:
Sounds good. I was on the fence.